07/src/academy/pocu/comp2500samples/w07/createentity/Program.java

```diff
@@ -0,0 +1,92 @@
+package academy.pocu.comp2500samples.w07.createentity;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class Program {
+    public static void main(String[] args) {
+        GameObject batman = loadGameObjectOrNull("Batman");
+        batman.update();
+
+        System.out.println();
+
+        GameObject tree = loadGameObjectOrNull("Tree");
+        tree.update();
+
+        System.out.println();
+
+        GameObject scaryVampire = loadGameObjectOrNull("ScaryVampire");
+        scaryVampire.update();
+    }
+
+    private static GameObject loadGameObjectOrNull(String name) {
+        String filename = String.format("%s.txt", name);
+        Path playerFilePath = Paths.get(getClassPath(), filename);
+        File playerFile = new File(playerFilePath.toString());
+
+        if (!playerFile.isFile()) {
+            return null;
+        }
+
+        List<String> lines;
+
+        try
+        {
+            lines = Files.readAllLines(
+                    playerFilePath,
+                    StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            throw new RuntimeException(e);

```

Comment:
shouldn't this be return null as well? 

the func says it'll return null when there's an issue. so better to return null after some logging